src/components/blocks/SearchResultList.tsx

```diff
@@ -1,13 +1,19 @@
 import { ExternalLink } from "@components/ExternalLink";
 import SearchResult from "./SearchResult";
 import { TMatchedFamily } from "@types";
+import { DOCUMENT_CATEGORIES } from "@constants/documentCategories";
+import Button from "@components/buttons/Button";
 
 type TProps = {
-  category?: string;
+  category?: (typeof DOCUMENT_CATEGORIES)[number];
```

Comment:
:grin: I would like this to be its own type ... say `CategoryList` ?

Reply:
I will circle back to this later